src/components/dashboard/main-content/course-enrollments/CourseEnrollments.jsx

```diff
@@ -110,6 +110,13 @@ const CourseEnrollments = ({ children }) => {
     );
   }
 
+   const handleOnClose = () => {
+    // 1) setShowCancelledAssignmentsAlert(false);
```

Comment:
I'm wondering if we should wait to hide the alert from the UI until the API call has successfully resolved? Kinda on the fence here.